### PR TITLE
some more fixes for Postfix on Ubuntu 20.04

### PR DESCRIPTION
**Pull request by enoch85**
add back setting password
change date 
add Author
try to fix formatting

@flurdy Please have a look

---
This change is [Reviewable](https://reviewable.io/reviews/flurdy/flurdy.com-docs/16)

### CODE REVIEW

File: postfix/edition15.html [hunk 1]
```diff
@@ -60,13 +60,14 @@ <h3>
 <h6><a href="#editions">15th edition</a></h6>
 <h6>
 	Author <a href="#author">Ivar Abrahamsen</a>
+	Author <a href="#author">Daniel Hansson</a>
```
Comment (flurdy) [inline postfix/edition15.html:63]: :) 
I very much appreciate the effort but I think perhaps I will add a proper contributors section. There have been a few contributors over the years so  I'll do that in a separate commit/PR  If you have a link twitter/website/github you prefer to link to then let me know.

Comment (flurdy) [reply postfix/edition15.html:63]: Of course, if it turns out you rewrite the entire howto then maybe we need to revisit the main author part!

File: postfix/edition15.html [hunk 2]
```diff
@@ -60,13 +60,14 @@ <h3>
...
 </h6>
 <h6>
 	License:
 	<a href="http://flurdy.com/docs/license/respect">Respect</a> (CC by-sa)
 <!-- 	on top of <a href="http://creativecommons.org/licenses/by-sa/2.5/">Creative Commons by-sa</a>. -->
 </h6>
-<h6>Last Update: 2020-05-08</h6>
+<h6>Last Update: 2020-12-04</h6>
```
Comment (flurdy) [inline postfix/edition15.html:70]: Ill revert a local commit I had to do this. Thanks :)

File: postfix/edition15.html (hunk 1 above)
Comment (enoch85) [reply postfix/edition15.html]: Fixed!

File: postfix/edition15.html (hunk 2 above)
Comment (enoch85) [reply postfix/edition15.html:69]: I don't understand? Can't you not just merge your local branch?

Comment (flurdy) [reply postfix/edition15.html:69]: Ah it wasn't needed anymore. All it had was the date update.

**Review by flurdy — APPROVED** [review]

**Comment by flurdy** [comment]
Ready for me to merge?

**Comment by enoch85** [comment]
Yes, please merge.

**Pull request merged**